website/src/components/index-page.tsx

```diff
@@ -1,24 +1,22 @@
 import { ReactElement } from 'react';
-import dynamic from 'next/dynamic';
 import { HeroGradient, HeroIllustration, NPMBadge } from '@theguild/components';
 import gqlCodegenCover from '../../public/assets/illustrations/gql-codegen-cover.svg';
 import gqlGenerateCodeIllustration from '../../public/assets/illustrations/gql-generate-code-illustration.svg';
 import gqlWatchForChangesIllustration from '../../public/assets/illustrations/gql-watch-for-changes-illustration.svg';
 
-const LiveDemo = dynamic(() => import('@/components/live-demo/LiveDemo'), { ssr: false });
-
 export function IndexPage(): ReactElement {
   return (
     <>
       <HeroGradient
-        title="Generate code from your GraphQL schema"
-        description="Generate code from your GraphQL schema and operations with a simple CLI"
-        link={{
-          href: '/docs/getting-started/installation',
-          children: 'Try It Now',
-          title: 'Get started with GraphQL Code Generator',
-        }}
-        version={<NPMBadge name="@graphql-codegen/cli" />}
```

Comment:
This does not provide any value here

Reply:
agreed. thanks for removing. We have many packages so doens't make sense to have this